test/unit/pdf_find_controller_spec.js

```diff
@@ -92,7 +104,63 @@ describe('pdf_find_controller', function() {
         expect(pdfFindController.pageMatches[i].length)
           .toEqual(matchesPerPage[i]);
       }
-      done();
+      callback();
+    });
+  }
+
+  it('performs a normal search', function(done) {
```

Comment:
Could you please also add *another* test, basically identical to this one, but with `findPrevious: true` set such that that code-path is covered by tests as well.

Reply:
Done. I have also extended the other unit tests to actually verify that the selected match is correct.